src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,11 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => user && (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
 );
```

Comment:
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => user && (
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
);
```

use desructuring for name and email

Reply:
@tiserett Is it worth of it? I'll explain. Destructuring objects is used to shorten code, make it more readable. Let's ompare my current solution:
```
export const UserInfo: React.FC<Props> = ({ user }) => user && (
  <a className="UserInfo" href={`mailto:${user.email}`}>
    {user.name}
  </a>
);
```
with proposed bu you:

```
export const UserInfo: React.FC<Props> = ({ user }) => {
  if (!user) {
    return null;
  }

  const { name, email } = user;

  return (
    <a className="UserInfo" href={`mailto:${email}`}>
      {name}
    </a>
  );
};
```

I like using destructuring, but in this case it less preferable. What do you think?

Reply:
u right, haven't mentioned you check if user exists

Reply:
don't need to render UserInfo if you haven’t a user, so you can expect that the user always will be